src/main/kotlin/io/github/enyason/client/Replicate.kt

```diff
@@ -0,0 +1,21 @@
+package io.github.enyason.client
+
+import io.github.enyason.domain.models.Prediction
+import io.github.enyason.predictable.Predictable
+import io.github.enyason.predictions.PredictionsApi
+
+interface Replicate {
+    companion object {
+        fun client(token: String): Replicate {
+            return ReplicateClient(predictionAPI = PredictionsApi(token))
+        }
```

Comment:
We should have an overloaded client function that accepts ReplicateConfig as a param